R/data_relocate.R

```diff
@@ -10,6 +11,14 @@
 #' @param safe If `TRUE`, will disregard non-existing columns.
 #' @inheritParams data_rename
 #'
+#' @inherit data_rename seealso
+#'
+#' @details `data_relocate()` will reorder columns to specific positions,
```

Comment:
Super minor: Maybe better to have one line per individual function? Given how similar the function names, it's easy to miss the change in function name in the second sentence.

Reply:
Do you mean separating these two sentences with a new line?

Reply:
Yes, like so:

```r
#' @details 
#'
#' `data_relocate()` will reorder columns to specific positions, indicated by `before` or `after`. 
#'
#' `data_reorder()` will instead always move selected columns to the beginning of the data frame .
```